src/test/java/org/adridadou/ethereum/propeller/backend/EthereumTest.java

```diff
@@ -143,6 +146,11 @@ public synchronized EthData constantCall(final EthAccount account, final EthAddr
         return localExecutionService.executeLocally(account, address, value, data);
     }
 
+    @Override
+    public List<EventData> logCall(SolidityEvent eventDefiniton, EthAddress address, String... optionalTopics) {
+        return null;
```

Comment:
We should have an implementation here.
The reason being that we don't want to have a different behavior if you use EthereumTest for your tests or if you run in production.

The implementation should be fairly simple and doesn't need to be very efficient. The reason being that you will never have a lot of blocks and they are all local anyway.

Reply:
once this is merged, I'll copy this implementation into eth-propeller-ethj so it can be used for testing